packages/app/templates/extensions/projects/flow_action/shopify.extension.toml.liquid

```diff
@@ -1,6 +1,5 @@
 name = "{{ name }}"
```

Comment:
Will these be renamed as `shopify.extensions.toml` as part of this PR?

Reply:
No, per my last conversation with Meredith, we'll continue using the singular name for now. But we can revisit that after editions